cirq-core/cirq/qis/clifford_tableau.py

```diff
@@ -306,6 +307,8 @@ def _row_to_dense_pauli(self, i: int) -> 'cirq.DensePauliString':
             represents the effective single Pauli operator on that qubit. The
             overall phase is captured in the coefficient.
         """
+        from cirq.ops.dense_pauli_string import DensePauliString
```

Comment:
we try to avoid inline imports, instead use `from cirq import ops` on the top and here you can use `ops.DensePauliString` to avoid circular import issues.

Reply:
Hmmm, if we move it to the top, I still encounter the error:

```
ImportError while loading conftest '/Users/ybc/Documents/github/MyCirq/cirq-core/cirq/conftest.py'.
cirq-core/cirq/__init__.py:30: in <module>
    from cirq import (
cirq-core/cirq/qis/__init__.py:24: in <module>
    from cirq.qis.clifford_tableau import CliffordTableau
cirq-core/cirq/qis/clifford_tableau.py:18: in <module>
    from cirq import ops
cirq-core/cirq/ops/__init__.py:21: in <module>
    from cirq.ops.clifford_gate import (
cirq-core/cirq/ops/clifford_gate.py:51: in <module>
    ) -> qis.CliffordTableau:
E   AttributeError: module 'cirq.qis' has no attribute 'CliffordTableau'
```
Looks like circular import again.